plan/expression_rewriter.go

```diff
@@ -134,6 +145,8 @@ func (er *expressionRewriter) Enter(inNode ast.Node) (ast.Node, bool) {
 		}
 	case *ast.SubqueryExpr:
 		return er.handleScalarSubquery(v)
+	case *ast.ParenthesesExpr:
+		er.asScalar = true
```

Comment:
If original 'asScalar' is not true, it will be set to true after visiting `ParenthesesExpr`